Snakemake/workflow/scripts/tsparam.py

```diff
@@ -0,0 +1,7 @@
+# parameters for tsinfer --- change accordingly
```

Comment:
Will this one leave in the /scripts folder? And then the other python file will find it?

Reply:
They have to be in the same folder (if they aren't that was my mistake)